### ISSUE TITLE
Salma and Youssef voices

### ISSUE THREAD

**Issue by rryan22** [GoAnimate-Wrapper/GoAnimate-Wrapper#47]
Is there Salma and Youssef Arabic voices on GoAnimate Wrapper?

**Comment by Windows81** [comment]
No, there are not.  If it's on Wrapper Offline, I might add them.

**Comment by rryan22** [comment]
How can this add Salma And Youssef Acapela Arabic voices to GA Wrapper? So this means it speaks in Arabic Language like I did on Xtranormal over the years. Is this gonna come?

**Comment by Windows81** [comment]
Probably.  I have no strict intention to آد ذيز فايساس.
